docs/central-upgrade.rst

```diff
@@ -74,6 +77,46 @@ You'll be asked to confirm the removal of all dangling images. Agree by typing t
 
     docker-compose up -d
 
+.. _central-upgrade-2022.3:
+
+Upgrading to Central v2022.3
+-----------------------------
+
+In v2022.3, we added Content Security Policy reporting. If you have disabled or customized Sentry, then you will need to modify ``files/nginx/odk.conf.template``. See our documentation about :ref:`configuring Sentry <central-install-digital-ocean-sentry>` to learn more about the specific changes that you need to make.
+
+.. _central-upgrade-1.5.3:
+
+Upgrading to Central v1.5.3
```

Comment:
This section was adapted from the forum post: https://forum.getodk.org/t/odk-central-v1-5/37882/4